src/components/Head/Head.js

```diff
@@ -27,7 +27,7 @@ function Head({ title, description, keywords }) {
       <link rel="apple-touch-icon" sizes="180x180" href="/favicons/apple-touch-icon.png" />
       <link rel="icon" type="image/png" sizes="32x32" href="/favicons/favicon-32x32.png" />
       <link rel="icon" type="image/png" sizes="16x16" href="/favicons/favicon-16x16.png" />
-      <link rel="manifest" href="/favicons/site.webmanifest" />
+      <link rel="manifest" href="/favicons/site.webmanifest" crossorigin="use-credentials" />
```

Comment:
```suggestion
      <link rel="manifest" href="/favicons/site.webmanifest" crossOrigin="use-credentials" />
```
just to correct React DOM friendly property name
![Screen Shot 2021-03-21 at 4 24 10 PM](https://user-images.githubusercontent.com/16929537/111919751-f4e45e80-8a61-11eb-8e57-85c6b2947801.png)